README.md

```diff
@@ -1,7 +1,7 @@
-Ethiopian repos by @nathenapse
+Useful Projects (Repos) related to Ethiopia
```

Comment:
why did u change the title??

Reply:
I thought it would be good not to mention single user to motivate others to contribute.